airbyte-integrations/connectors/source-postgres/Dockerfile

```diff
@@ -12,9 +12,16 @@ FROM airbyte/integration-base-java:dev
 
 WORKDIR /airbyte
 
+# Add the Datadog Java APM agent
+ADD https://dtdg.co/latest-java-tracer dd-java-agent.jar
+
 ENV APPLICATION source-postgres
 
+ENV DD_SERVICE: "source-postgres"
+ENV DD_VERSION: 1.0.35-dd
```

Comment:
This should probably be set to the real version number/revert the change to the label below before merging.  We probably also need to update whatever script updates the release number to contain the `DD_VERSION` as a line that also needs to be modified when releasing a new version.